apps/daffio/src/app/core/sidebar/services/sidebar.service.ts

```diff
@@ -71,6 +71,6 @@ export class DaffioSidebarService extends DaffSidebarService {
     @Inject(SERVER_SAFE_BREAKPOINT_OBSERVER) private breakpointObserver: BreakpointObserver,
     private routerData: DaffRouterDataService<DaffioRoute['data']>,
   ) {
-    super(DAFFIO_NAV_SIDEBAR_ID);
+    super(DAFFIO_MARKETING_NAV_SIDEBAR_ID);
```

Comment:
@griest024 Not sure what I should've done here

Reply:
reverted